apps/upgrade-single-node/create.py

```diff
@@ -0,0 +1,56 @@
+import json
+import time
+
+import weaviate
+import weaviate.classes as wvc
+
+sphere_file = "sphere.10k.jsonl"
+
+sizes = [10, 100, 1000, 10000, 10, 10, 10, 10, 50]
+
+
+def _import():
+    client = weaviate.connect_to_local()
+    client.collections.delete(["MultiTenantPerformanceTest"])
+    collection = client.collections.create(
+        "MultiTenantPerformanceTest",
+        multi_tenancy_config=wvc.config.Configure.multi_tenancy(enabled=True),
+        properties=[
+            wvc.config.Property(name="url", data_type=wvc.config.DataType.TEXT),
+            wvc.config.Property(name="title", data_type=wvc.config.DataType.TEXT),
+            wvc.config.Property(name="raw", data_type=wvc.config.DataType.TEXT),
+            wvc.config.Property(name="sha", data_type=wvc.config.DataType.TEXT),
+        ],
+        vector_index_config=wvc.config.Configure.VectorIndex.flat(quantizer=wvc.config.Configure.VectorIndex.Quantizer.bq(cache=True))
+    )
+
+
+    num_tenants = len(sizes)
+    print(sizes)
+    for j in range(num_tenants):
+        name = f"Tenant{j}"
+        collection.tenants.create(name)
+
+        with collection.with_tenant(name).batch.dynamic() as batch:
+            with open(sphere_file) as jsonl_file:
+                start = time.time()
+                for i, jsonl in enumerate(jsonl_file):
+                    if i >= sizes[j]:
+                        break
+
+                    json_parsed = json.loads(jsonl)
+                    batch.add_object(
+                        properties={
+                            "url": json_parsed["url"],
+                            "title": json_parsed["title"],
+                            "raw": json_parsed["raw"],
+                            "sha": json_parsed["sha"],
```

Comment:
can we do this with some dummy data to not have to save the sphere dataset? Shouldn't play a role

Reply:
i will think about it, but the idea is large data to create a snapshot